recommender/recommender.py

```diff
@@ -4,7 +4,9 @@
 """
 import hashlib
 import json
+import lxml.etree as etree
 import pkg_resources
+import xml.dom.minidom as md
```

Comment:
Why do you need to use `xml.dom.minidom`? Why not just use `lxml`? `lxml` is faster and more widely used.